python/tests/test-4-qfunction.py

```diff
@@ -282,7 +302,8 @@ def test_412(ceed_resource):
 
     q = 8
 
-    u_array = np.zeros(q * size, dtype="float64")
+    u_array = np.zeros(q * size,
+                       dtype=libceed.scalar_types[libceed.lib.CEED_SCALAR_TYPE])
```

Comment:
This feels a little clunky. How about adding a `ceed.scalar_type()` helper function to do this? Eventually I would see us doing this per object.

Reply:
I thought we were wanting to avoid this, as with the `CeedGetScalarType` discussion above, if it would later be changed to per-object anyway?  (The difference being that the Julia interface required something like this to work in the interim.)

Reply:
We could go directly to a per device version if you prefer. I'm just objecting to repeating `libceed.scalar_types[libceed.lib.CEED_SCALAR_TYPE]` over and over and over all over the Python test suite

Reply:
Please check out ebfc85f15683e73b56e841e98b4bfbe5c12304c5 and see if this is what you had in mind.